StortSpelProjekt/Game/src/Gamefiles/EnemyFactory.cpp

```diff
@@ -201,6 +200,7 @@ Entity* EnemyFactory::Add(const std::string& entityName, EnemyComps* comps)
 	t->SetRotationY(comps->rot.y);
 	t->SetRotationZ(comps->rot.z);
 	t->SetVelocity(comps->movementSpeed * 0.5);
+	t->UpdateWorldMatrix();
```

Comment:
Does this fix the "spawn huge at origo" bug?

Reply:
me think so maybe